apps/hpc-ftsadmin/src/app/app.tsx

```diff
@@ -2,21 +2,22 @@ import {
   BaseStyling,
   C,
   CLASSES,
-  dataLoader,
   dialogs,
   styled,
   ThemeProvider,
+  useDataLoader,
 } from '@unocha/hpc-ui';
 import { useEffect, useState } from 'react';
+import { MdAdd } from 'react-icons/md';
 import { Outlet } from 'react-router';
-import { ToastContainer } from 'react-toastify';
-import env, { type Environment } from '../environments/environment';
+import { Slide, ToastContainer } from 'react-toastify';
+import env, { type Environment } from '../environments/environment.prod';
```

Comment:
This change caused a problem on my local with error that config cannot be found - `Error: Unable to get config (404): Not Found`. The environment files are being replaced in `project.json` configuration with the correct file (depending on environment), so you should revert the change to:
```suggestion
import env, { type Environment } from '../environments/environment';
```